posix/include/sof/lib/dma.h

```diff
@@ -97,6 +97,9 @@ enum dma_cb_status {
 	DMA_CB_STATUS_END,
 };
 
+#define SOF_DMA_CB_STATUS_RELOAD	DMA_CB_STATUS_RELOAD
+#define SOF_DMA_CB_STATUS_END		DMA_CB_STATUS_END
+
```

Comment:
Apologies @andyross , I was testing with all AMD/MTK/NXP/Intel targets in upstream, but right, definitions not used in upstream were very likely missed.